dataprofiler/profilers/categorical_column_profile.py

```diff
@@ -228,6 +277,10 @@ def is_match(self) -> bool:
             is_match = True
         return is_match
 
+    def _get_categories(self, df_series):
```

Comment:
slight refactor which moves the get to its own function. important to allow usage of `__calculations` and to accurately use the `timeit` functionality.